src/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -114,43 +108,58 @@ export class ObjectsTable extends Component {
 
   fetchCounts = async () => {
     const { queryText, visibleTypes } = parseQuery(this.state.activeQuery);
-    const type = INCLUDED_TYPES.filter(
+
+    const filteredTypes = INCLUDED_TYPES.filter(
       type => !visibleTypes || visibleTypes.includes(type)
     );
 
+    // These are the saved objects visible in the table.
+    const filteredSavedObjectCounts = await getSavedObjectCounts(
+      this.props.$http,
+      filteredTypes,
+      queryText
+    );
+
+    const exportAllOptions = [];
+    const exportAllSelectedOptions = {};
+
+    Object.keys(filteredSavedObjectCounts).forEach(id => {
+      // Add this type as a bulk-export option.
+      exportAllOptions.push({
+        id,
+        label: `${id} (${filteredSavedObjectCounts[id] || 0})`,
+      });
+
+      // Select it by defayult.
+      exportAllSelectedOptions[id] = true;
+    });
+
+    // These are all the saved objects that exist.
     const savedObjectCounts = await getSavedObjectCounts(
       this.props.$http,
-      type,
+      INCLUDED_TYPES,
```

Comment:
Why was this changed from `filteredTypes ` to all types?

Reply:
It's used to populate the filter dropdown in the table. Originally, filtering on a type would reduce the counts of the non-selected types to 0. I'll add a comment.